vassal-app/src/main/java/VASSAL/build/module/ChessClockControl.java

```diff
@@ -419,7 +419,7 @@ else if (SHOW_DAYS.equals(key)) {
       return showDays;
     }
     else if (ALLOW_RESET.equals(key)) {
-      return ALLOW_RESET;
+      return ((Boolean)allowReset).toString();
```

Comment:
The right way here would be either `String.valueOf(allowReset)` or `Boolean.toString(allowReset)`.

Reply:
done